### PR TITLE
dqm for triplemuon_5_3_3_dz in master br.

### PR DESCRIPTION
**Pull request by gkaratha**
Dear @mtosi @davidlange6 ,

I did the changes that were asked. And also the PR in master branch as well. Now there is one PR in master, one in 9_3_X and one in 9_2_X(backport of 9_3_X). For any other comment i am at your disposal. Thanks.

Best,
george

### CODE REVIEW

**Comment by mtosi** [comment]
@cmsbuild, please test

File: DQMOffline/Trigger/plugins/TopMonitor.cc [hunk 1]
```diff
@@ -675,6 +678,21 @@ void TopMonitor::analyze(edm::Event const& iEvent, edm::EventSetup const& iSetup
...
 
+    bool allpairs=false;
+  if (nmuons_>2) {
+    float mumu_mass;
+    for (unsigned int idx=0; idx<muons.size(); idx++){
+      for (unsigned int idx2=idx+1; idx2<muons.size(); idx2++){
+         //compute inv mass of two different leptons
+          mumu_mass=(muons[idx2].p4() + muons[idx2].p4()).M();
+           if (mumu_mass<invMassLowercut_ || mumu_mass>invMassUppercut_) allpairs=true;
+                       }
```
Comment (dmitrijus) [inline DQMOffline/Trigger/plugins/TopMonitor.cc]: please please indent your code
this is confusing and unreadable

File: DQMOffline/Trigger/python/SusyMonitoring_Client_cff.py [hunk 2]
```diff
@@ -98,6 +98,23 @@
 
         ),
 )
+triple_muon_mupt_efficiency = DQMEDHarvester("DQMGenericClient",
+    subDirs        = cms.untracked.vstring("HLT/SUSY/SOS/TripleMu/MuonPt/*"),
+    verbose        = cms.untracked.uint32(0), # Set to 2 for all messages 
+    resolution     = cms.vstring(),
+    efficiency     = cms.vstring(
+"effic_muPt_1       'efficiency vs muon pt; muon pt [GeV]; efficiency' muPt_1_numerator       muPt_1_variableBinning_denominator",
```
Comment (dmitrijus) [inline DQMOffline/Trigger/python/SusyMonitoring_Client_cff.py]: same as above

**Review by dmitrijus — CHANGES_REQUESTED** [review]

File: DQMOffline/Trigger/python/SusyMonitoring_Client_cff.py (hunk 2 above)
Comment (gkaratha) [reply DQMOffline/Trigger/python/SusyMonitoring_Client_cff.py]: Hi @dmitrijus ,

Ok fixed thanks

**Comment by dmitrijus** [comment]
+1

**Comment by mtosi** [comment]
@davidlange6 @dmitrijus 
can we proceed w/ the merge ?
thanks !

**Comment by davidlange6** [comment]
merge

**Pull request merged**